### PR TITLE
Refactor code, fix comments

### PR DESCRIPTION
**Pull request by Joyfal**
## Checklist

- [x] Did you write tests for your code?
- [x] Did all the tests pass locally?

## Trello Ticket
https://trello.com/c/BxpiGJdk/66-update-code-and-comments

## Short Description
See ticket.

## Reviewer Notes

Anything special you want the reviewer to know?

### CODE REVIEW

File: src/modules/ChatRoom.js [hunk 1]
```diff
@@ -161,39 +161,34 @@ class ChatRoom extends React.Component {
...
         if (!qs.empty) {
-          qs.forEach((doc) => {
-            this.setState({
-              roomId: doc.id,
-            });
-          });
+          const room = qs.docs[0];
+          return room.id;
+        } else {
+          return "";
```
Comment (macedo22) [inline src/modules/ChatRoom.js:188]: Just a suggestion - maybe this should be `return null` instead. What do you think?

[the remark this answers is not in the document]
Comment (macedo22) [reply src/modules/ChatRoom.js:188]: I don't think it makes a practical difference, but just seems consistent with the concept of "no room found"

[the remark this answers is not in the document]
Comment (macedo22) [reply src/modules/ChatRoom.js:188]: Whether we go with the empty string or `null`, I think we should also add `roomId` as a field in the state, e.g.:
```
this.state = {
      message: "",
      messages: [],
      user: user,
      otherUser: null,
      users: [],
      keyword: "",
      roomName: "Chat Room",
      roomId: null, // or ""    <----- add this
      loading: true,
    };
```

**Review by macedo22 — COMMENTED** [review]
Two really tiny suggestions, feel free to take or leave. There is still a good amount that I think needs to be fixed with the mocking, but I think it's outside the scope/nature of this PR and I can handle it in another one.

**Review by macedo22 — APPROVED** [review]
I'm going to pre-approve it so it can be merged. Feel free to take or leave the two suggestions above

File: src/modules/ChatRoom.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Joyfal) [reply src/modules/ChatRoom.js:188]: good point, adding that.

**Review by macedo22 — APPROVED** [review]

**Pull request merged**